src/System.Private.CoreLib/shared/System/Globalization/DateTimeFormatInfoScanner.cs

```diff
@@ -204,14 +205,14 @@ internal static int SkipWhiteSpacesAndNonLetter(string pattern, int currentIndex
         // A helper to add the found date word or month postfix into ArrayList for date words.
         //
         // Parameters:
-        //      formatPostfix: What kind of postfix this is.  
+        //      formatPostfix: What kind of postfix this is.
         //          Possible values:
         //              null: This is a regular date word
         //              "MMMM": month postfix
         //      word: The date word or postfix to be added.
         //
         ////////////////////////////////////////////////////////////////////////////
-        internal void AddDateWordOrPostfix(string formatPostfix, string str)
+        internal void AddDateWordOrPostfix(string? formatPostfix, string str)
```

Comment:
Shouldn't be nullable. (I know blame me 😆)

Reply:
Actually, disregard, there is nothing preventing it from being null.

Reply:
actually your tool did it correctly :P we never throw when null is passed and line 300 actually may pass null :smile:

Reply:
See, sometimes it is worth assuming if there is no validation that it can be null 😆

Reply:
im still not sure if the tool should be checking for "can be null" or rather "is code expecting null" :smile: